docs/getting-started/no-docker-networking/GettingStarted.md

```diff
@@ -140,5 +157,5 @@ For example:
 
     calicoctl node --ip=172.17.8.101 --ip6=fd80:24e2:f998:72d7::1
 
```

Comment:
It would be nice to have a link to the Advanced Network Policy file in here as well.